### PR TITLE
Add option for using K8s secrets resources to KBS

### PR DESCRIPTION
**Pull request by bpradipt**
This PR allows configuring K8s secrets as KBS resources

### CODE REVIEW

File: config/manifests/bases/kbs-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - kbs-operator
+  - attestation-service
+  - rvps
+  links:
+  - name: Kbs Operator
+    url: https://kbs-operator.domain
+  maintainers:
+  - email: cncf-ccontainers-maintainers@lists.cncf.io
+    name: Pradipta Banerjee
+  - email: ' cncf-ccontainers-maintainers@lists.cncf.io'
```
Comment (lmilleri) [inline config/manifests/bases/kbs-operator.clusterserviceversion.yaml:47]: is the space before cncf needed?

File: controllers/kbsconfig_controller.go [hunk 2]
```diff
@@ -359,6 +371,20 @@ func volumesToVolumeMounts(volumes []corev1.Volume) []corev1.VolumeMount {
 	return volumeMounts
 }
 
+// Method to add volumeMounts for KBS under custom directory
+func volumesToVolumeMountsCustom(volumes []corev1.Volume, mountPath string) []corev1.VolumeMount {
```
Comment (lmilleri) [inline controllers/kbsconfig_controller.go:375]: Can we avoid having 2 functions doing the same thing? I think you can call `volumesToVolumeMountsCustom` with "/etc" mount path for the `kbsEtcVolumes`

**Comment by lmilleri** [comment]
Can you please add a short section in the README about how to add the resources?

File: config/manifests/bases/kbs-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (bpradipt) [reply config/manifests/bases/kbs-operator.clusterserviceversion.yaml]: Not needed. I have updated it

**Comment by lmilleri** [comment]
Thx, it looks good to me

**Review by lmilleri — APPROVED** [review]

**Pull request merged**